docs-src/sidebars.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
         "kognic-io/annotations",
        ]},
       "kognic-io/working_with_scenes_and_inputs",
-      "kognic-io/calibrations",
+      {"Calibrations": [
+        "kognic-io/calibrations/overview",
+        "kognic-io/calibrations/lidars",
+        "kognic-io/calibrations/cameras-custom",
```

Comment:
Standard should maybe come before custom here. I think it would make sense if the standard was above custom in the menu.